pom.xml

```diff
@@ -47,6 +47,11 @@
         <tag>${scmTag}</tag>
     </scm>
     <dependencies>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>lib-durable-task</artifactId>
+            <version>${lib-durable-task.version}</version>
```

Comment:
No reason to use a POM version unless you are referring to it at least twice.

Reply:
Only one instance used now in c658ad81067e362c9fb0c9d51763d3232a97a6f2